### PR TITLE
Add ldap username field option. Default field is 'cn'

### PR DESCRIPTION
**Pull request by bananayong**

### CODE REVIEW

**Comment by bananayong** [comment]
Hi~

Thank you for your awesome stuff. stf is really great job.

I am trying to install stf in my workspace. But, I have a problem when applying ldap auth.
I think that my workspace ldap settings are uncommon. cn field is a array. ex) ['employee ID', 'employee name']

So, I would like to add an option which can select ldap field for stf username.

Please check this PR.
thank you in advance

File: lib/cli.js [hunk 1]
```diff
@@ -360,6 +360,10 @@ program
     , 'LDAP search field (or $LDAP_SEARCH_FIELD)'
     , String
     , process.env.LDAP_SEARCH_FIELD)
+  .option('--ldap-username-field <scope>'
```
Comment (sorccu) [inline lib/cli.js]: Small thing but could you replace `<scope>` with `<username>`?

**Comment by sorccu** [comment]
Hey thanks! Don't mind about the Travis failure, it's a bit screwed up right now. I'll merge this if you can fix the small thing I commented on separately.

Also thanks for keeping to the existing coding style!

**Comment by bananayong** [comment]
@sorccu Thanks quick review!
I fixed typo

Thank you for merging!!

**Comment by sorccu** [comment]
Thanks!

**Pull request merged**